fdbclient/Schemas.cpp

```diff
@@ -24,26 +24,26 @@
 const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
 {
    "cluster":{
-	  "storage_wiggler": {
+       "storage_wiggler": {
          "primary": {
-          	"last_round_start": 1,
-			"last_round_finish": 1,
-			"smoothed_round_duration": 1,
+          	"last_round_start": "%c %z",
+			"last_round_finish": "%c %z",
```

Comment:
Let's add `_datetime` to the start and finish field but also add a `_timestamp` field for each which is integer epoch seconds, which matches the other `_timestamp` fields we have in status.   JSON status is meant to be machine readable, integer epochs is probably the easier thing to read in code but having the human readable field is good too.

Reply:
Also, the model in this document is actually to provide a type schematic for the JSON document, there is no validation done other than datatype so the contents of the string do not matter.  `"%c %z"` is a string so it is technically acceptable but I think it would be better to use a string which is an example of this timestamp format instead.  For example, elsewhere in the doc is
```
      "cluster_controller_timestamp":1415650089,
```
as an example of integer epoch seconds which is what is found in this integer timestamp field.